deployment/workspaces/daniel.naab/main.tf

```diff
@@ -1,25 +1,33 @@
-variable "cf_api_url" { default = "https://api.fr.cloud.gov" }
-
-variable "cf_env" { default = "dev" }
-variable "cf_org" { default = "sandbox-gsa" }
-variable "cf_space" { default = "taylor.zajicek" }
-variable "cf_rds_strapi_db_service_plan" { default = "shared-psql" }
-variable "cf_s3_strapi_image_plan" { default = "basic-public-sandbox" }
-variable "cf_s3_frontend_plan" { default = "basic-public-sandbox" }
-
-variable "cf_strapi_logingov_key" { default = "" }
-
-variable "cf_strapi_logingov_issuer" {
-  type = map
-  default = {
-    dev = "urn:gov:gsa:openidconnect.profiles:sp:sso:gsa:ai_experience"
+terraform {
+  required_providers {
+    cloudfoundry = {
```

Comment:
I encountered a strange (to me) issue that required setting this in both the workspace, and the module. It seems odd, but Terraform doesn't understand this is a cloudfoundry-community provider otherwise.